contracts/cw-proposal-single/src/proposal.rs

```diff
@@ -118,6 +124,7 @@ impl Proposal {
     /// Sets a proposals status to its current status.
     pub fn update_status(&mut self, block: &BlockInfo) {
         self.status = self.current_status(block);
+        self.last_updated = block.time
```

Comment:
Think we need to do this in `execute_close` and `execute_execute` as well if we want to count those as "updates".